ToolsTest/Volodymyr/xUnit.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Tools;
+
+using Xunit;
+using Assert = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
+using Tools.Models;
+
+
+namespace ToolsTest.Volodymyr
+{
+    public class xUnit
+    {
+        [Fact]
+        public void TryReplaceText()
+        {
+            // Arrange
+            Article article = new Article { Text = "this is old text in the article" };
+            string newText = "new text";
+            string oldText = "old text";
+
+            // Act
+            bool result = article.TryReplaceText(oldText, newText);
+
+            // Assert
+            Assert.IsTrue(result);
+            Assert.Equals("this is new text in the article", article.Text);
```

Comment:
This test fails here because of conflict between `Equals()` method from MSTest and `Equal()` method from xUnit.
Remove `using Assert = ...` (see previous comment), then fix method names (from `Equals()` to `Equal()` and from `IsTrue()` to `True()`), and this issue will be fixed.

However my congratulations, your test have found the bug in my code !)
This test will fail until I will fix the error. If you want you can fix it but generally it is not necessary as it is not your code.